wv2util/MainWindow.xaml.cs

```diff
@@ -336,5 +355,90 @@ private void WatchForChangesTimer_Elapsed(object sender, ElapsedEventArgs e)
             }
             m_previousHostAppEntries = currentHostAppEntries;            
         }
+
+        private string newsLink = "https://api.github.com/repos/MicrosoftEdge/WebView2Announcements/issues";
+        private async Task<string> GetNewsData()
+        {
+            var client = new HttpClient();
+            client.DefaultRequestHeaders.Accept.Clear();
+            client.DefaultRequestHeaders.Accept.Add(new System.Net.Http.Headers.MediaTypeWithQualityHeaderValue("application/vnd.github.v3+json"));
+            client.DefaultRequestHeaders.Add("User-Agent", ".NET Foundation Repository Reporter");
+
+            var response = client.GetAsync(newsLink).Result;

```

Comment:
Can you use await here instead of .Result?
```suggestion
            var response = await client.GetAsync(newsLink);
```